util/getGuildInfo.js

```
@@ -1,9 +1,16 @@
+	let uniqueMembers = new Map();
 		client.guilds.cache.each(guild => {
 			guildCount++
 			totalMembers += guild.memberCount;
+			guild.members.cache.each(member => {
+				if(!uniqueMembers.get(member.id)){
+					uniqueMembers.set(member.id, true);
+				}
+			})
 		});
-	return {guildCount: guildCount, totalMembers: totalMembers};
+	let uniqueMemberCount = uniqueMembers.size;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'uniqueMemberCount' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)